catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/BoundingBoxReader.java

```diff
@@ -130,40 +150,59 @@ private String createWkt(String[] lowerCornerPosition, String[] upperCornerPosit
+        return "POLYGON ((" + lowerCornerPosition[0] + SPACE + lowerCornerPosition[1] + ", "
                 + upperCornerPosition[0] + SPACE + lowerCornerPosition[1] + ", "
                 + upperCornerPosition[0] + SPACE + upperCornerPosition[1] + ", "
                 + lowerCornerPosition[0] + SPACE + upperCornerPosition[1] + ", "
                 + lowerCornerPosition[0] + SPACE + lowerCornerPosition[1] + "))";
     }
 
     /**
-     * @param coords
-     *            The latitude and longitude coordinates (in no particular order).
-     * @param axisOrder
-     *            The order that the axes are represented.
-     * @return The coordinates in LON/LAT order.
+     * @param coords The latitude and longitude coordinates (in no particular order).
+     * @return The coordinates in an array
      */
-    private String[] getCoordinates(String coords, CswAxisOrder axisOrder) {
+    private String[] getCoordinates(String coords) {
+        return coords.split(SPACE);
+    }
 
-        switch (axisOrder) {
-        case LAT_LON: {
-            /**
-             * We want to create WKT in LON/LAT order. Since the response has the coords in LAT/LON
-             * order, we need to reverse them.
-             */
-            return StringUtils.reverseDelimited(coords, SPACE.charAt(0))
-                    .split(SPACE);
-        }
-        default: {
-            /**
-             * We want to create WKT in LON/LAT order. Since this is the order of the coords in the
-             * response, we use them as-is.
-             */
-            return coords.split(SPACE);
+    private String convertToEPSG4326(String wkt, String crs) throws CswException {
+        try {
+            Geometry geometry = wktReader.read(wkt);
+            Geometry convertedGeometry = GeospatialUtil.transformToEPSG4326LonLatFormat(geometry,
+                    crs);
+
+            if (convertedGeometry != null && withinEPSG4326Bounds(convertedGeometry)) {
+                return WKT_WRITER_THREAD_LOCAL.get()
+                        .write(convertedGeometry);
+            } else {
+                throw new GeoFormatException();
+            }
+        } catch (ParseException | GeoFormatException e) {
+            LOGGER.debug("Unable to convert {} from {} to EPSG:4326", wkt, crs, e);
+            throw new CswException(String.format("Unable to convert %s from %s to EPSG:4326",
+                    wkt,
+                    crs));
         }
+    }
+
+    private boolean isEPSG4326(String crs) {
+        return (crs.contains("4326") || crs.contains("CRS84"));
+    }
+
+    private boolean withinEPSG4326Bounds(Geometry geometry) {
+        Coordinate[] coordinates = geometry.getCoordinates();
+        for (Coordinate coordinate : coordinates) {
+            // Longitude
+            if (coordinate.x <= -180 || coordinate.x >= 180) {
+                return false;
+            }
+            // Latitude
+            if (coordinate.y <= -90 || coordinate.y >= 90) {
```

Comment:
nice catch, took away `=` for these comparisons (as we should be able to have points there)